requirements.txt

```diff
@@ -0,0 +1,31 @@
+appdirs==1.4.0
```

Comment:
can this be slimmed down so that it has what the playlist scripts directly depends on?

looks like for now it may just be gmusicapi and future.